specification/loadtestservice/LoadTestService.Management/tspconfig.yaml

```diff
@@ -31,3 +35,23 @@ options:
     flavor: "azure"
     generate-test: true
     generate-sample: true
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.ResourceManager.LoadTesting"
+    flavor: azure
+    clear-output-folder: true
+    namespace: "{package-dir}"
+  "@azure-tools/typespec-java":
+    package-dir: "azure-resourcemanager-loadtesting"
+    namespace: "com.azure.resourcemanager.loadtesting"
+    flavor: azure
+    service-name: Azure Load Testing
```

Comment:
Please use `Load Test` here. It needs to follow what's in Swagger to avoid break.
(unfortunately you call yourself `LoadTest` in Swagger https://github.com/Azure/azure-rest-api-specs/blob/85a0ca2368ebf6a3f33d6831a6b72a04a616ae40/specification/loadtestservice/resource-manager/Microsoft.LoadTestService/loadtesting/stable/2022-12-01/loadtestservice.json#L4

Reply:
@weidongxu-microsoft Thanks for calling this out. I'll make the change. Could you please review the same for playwright service if that looks good since it would be in public preview for first time here?